### PR TITLE
support nests in tabs view

### PR DESCRIPTION
**Pull request by xithan**
Example: the tabs view of a card with content 
````
{{A | labeled; title: tab A}}
{{B | titled; title: tab B}}
````
renders two tabs with the given titles as tab buttons and uses the given views for the tab panes.

### CODE REVIEW

File: card/mod/01_core/set/all/collection.rb [hunk 1]
```diff
@@ -160,28 +181,41 @@ def set_search_params_variables! hash
...
+            item.process_chunk { |options| prepare_nest options }
+          else
+            nest(Card.fetch(item, :new=>{}), i_args)
+          end
+        else
+          ''
+        end
       tab_panes += tab_pane( id, tab_content, active_tab )
     end
     tab_panel tab_buttons, tab_panes, args[:tab_type]
```
Comment (ethn) [inline card/mod/01_core/set/all/collection.rb]: this is getting a little hairy.  wish I had a good idea of how to simplify.

**Comment by ethn** [comment]
So is this smart about wrapping adjacent tabs somehow?

**Comment by ethn** [comment]
nevermind, I didn't read correctly.  I get it.  these are treated as items of a different card.

File: card/mod/01_core/set/all/collection.rb (hunk 1 above)
Comment (xithan) [inline card/mod/01_core/set/all/collection.rb]: a nest_to_url method could be a first step to make it more readable

**Pull request merged**